Gruntfile.js

```diff
@@ -383,6 +383,23 @@ module.exports = function (grunt) {
             src: ['**', '!hmda-pilot.zip']
           }
         ]
+      },
+      'codedeploy': {
```

Comment:
That is correct -- I made the 2nd grunt task because I assumed there would be instances where you want to do not want the CodeDeploy artifact and you only want the code from the build.